### PR TITLE
Various fixes

### PR DESCRIPTION
**Pull request by csomh**
This is a collection of commits which attempt to make conversion and updates work for some left-behind packages, and also to improve the stability of the scheduled update.

### CODE REVIEW

File: dist2src/worker/config.py [hunk 1]
```diff
@@ -23,19 +24,28 @@ def __init__(self):
 
         self._src_git_svc = None
         self._dist_git_svc = None
+        self._retries = Retry(
+            total=5,
+            backoff_factor=1,
+            status_forcelist=Retry.RETRY_AFTER_STATUS_CODES.union([404]),
+        )
```
Comment (lachmanfrantisek) [inline dist2src/worker/config.py:31]: I really like this object abstraction on top of retries.

File: dist2src/worker/config.py [hunk 2]
```diff
@@ -23,19 +24,28 @@ def __init__(self):
...
 
     @property
     def src_git_svc(self) -> PagureService:
         if self._src_git_svc is None:
             self._src_git_svc = PagureService(
-                instance_url=f"https://{self.src_git_host}", token=self.src_git_token
+                instance_url=f"https://{self.src_git_host}",
+                token=self.src_git_token,
+                max_retries=self._retries,
```
Comment (lachmanfrantisek) [inline dist2src/worker/config.py:39]: Now, when I see it, what about using `retry` or `retries` as an argument name for `PagureService`? (I can send a PR if we agree on it.)

**Review by lachmanfrantisek — COMMENTED** [review]
Nice improvements! A lot of code has been simplified. Thanks!

File: dist2src/worker/processor.py [hunk 3]
```diff
@@ -145,7 +145,16 @@ def update_project(self, project: PagureProject, conversion_tag: str):
...
 
     def cleanup(self):
-        if self.dist_git_dir:
-            shutil.rmtree(self.dist_git_dir, ignore_errors=True)
-        if self.src_git_dir:
-            shutil.rmtree(self.src_git_dir, ignore_errors=True)
+        """
+        Clean up the working directory.
+
+        This is safe, as long as no parallel work is done in this directory.
```
Comment (jpopelka) [inline dist2src/worker/processor.py:151]: Isn't this (parallel work) the case of more workers?

File: deployment/roles/deploy/templates/worker-sts.yml.j2 [hunk 4]
```diff
@@ -77,4 +77,4 @@ spec:
         accessModes: ["ReadWriteOnce"]
         resources:
           requests:
-            storage: 8Gi
+            storage: 10Gi
```
Comment (jpopelka) [inline deployment/roles/deploy/templates/worker-sts.yml.j2:80]: Shouldn't your "Clean up the entire working directory" commit fix the no-space-left issue?

**Review by jpopelka — APPROVED** [review]

File: dist2src/worker/processor.py (hunk 3 above)
Comment (csomh) [reply dist2src/worker/processor.py:151]: Nope, as each worker has it's own volume, and workers are configured to execute one task at a time.

File: dist2src/worker/config.py (hunk 2 above)
Comment (csomh) [reply dist2src/worker/config.py:39]: I've also thought about that name, but in the end I decided to stay consistent with the naming from [`requests.adapters.HTTPAdapter`](https://2.python-requests.org/en/master/api/#requests.adapters.HTTPAdapter). But I have no strong opinion on how to call this argument.

File: deployment/roles/deploy/templates/worker-sts.yml.j2 (hunk 4 above)
Comment (csomh) [reply deployment/roles/deploy/templates/worker-sts.yml.j2:80]: Not really, a few packages need even more space. I've seen [boost](https://git.centos.org/rpms/boost) eating up all the 8Gi, this is why I've increased the volume size to 10Gi.

**Comment by csomh** [comment]
recheck

**Comment by csomh** [comment]
recheck

File: dist2src/worker/config.py (hunk 2 above)
Comment (lachmanfrantisek) [reply dist2src/worker/config.py:39]: Ok, that consistency makes sense. I am not against now.

**Pull request merged**